### PR TITLE
Append internal web-server for checking service status.

### PR DESCRIPTION
**Pull request by ukolovda**
URL: `http://localhost:8088/status`
Now returns "starting" and "running" values.

HTTP port exposed in Dockerfile is 8088 and can be changed by command-line parameter `--http-port` or environment "PGTT_HTTPPORT". By default is 0 (disabled).

### CODE REVIEW

File: internal/pgtt_http/http_status.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		//Handler:        httpHandler,
+		ReadTimeout:    10 * time.Second,
+		WriteTimeout:   10 * time.Second,
+		MaxHeaderBytes: 1 << 20,
+	}
+	http.HandleFunc("/status", readyHandler)
+	logger.Infof("Starting HTTP server on port %d...", portNumber)
+	err := s.ListenAndServe()
+	if err != nil {
+		logger.WithError(err).Error("Error start HTTP server. %v")
```
Comment (pashagolub) [inline internal/pgtt_http/http_status.go:36]: `Error call has possible formatting directive %v`:
```suggestion
		logger.WithError(err).Error("Cannot start HTTP server")
```

File: internal/scheduler/tasks_test.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func TestExecuteTask(t *testing.T) {
 	assert.Error(t, et("Download", []string{"foo"}), "Invalid json")
 	assert.EqualError(t, et("Download", []string{`{"workersnum": 0, "fileurls": [] }`}),
 		"Files to download are not specified", "Download with empty files should fail")
-	assert.Error(t, et("Download", []string{`{"workersnum": 0, "fileurls": ["http://foo.bar"], "destpath": "" }`}),
+	assert.Error(t, et("Download", []string{`{"workersnum": 0, "fileurls": ["pgtt_http://foo.bar"], "destpath": "" }`}),
```
Comment (pashagolub) [inline internal/scheduler/tasks_test.go:44]:
```suggestion
	assert.Error(t, et("Download", []string{`{"workersnum": 0, "fileurls": ["http://foo.bar"], "destpath": "" }`}),
```

Comment (ukolovda) [reply internal/scheduler/tasks_test.go]: Sorry, mistake append. This file should not be changed.

**Comment by pashagolub** [comment]
Thanks for your help. I implemented the #379 PR based on your proposal. We'll proceed there.

**Pull request closed without merging**